aws_lambda_fsm/fsm.py

```diff
@@ -689,7 +691,7 @@ def dispatch(self, event, obj):
             # False
             # >>>
             #
-            if fence_token is 0 or fence_token is 0L:
+            if not isinstance(fence_token, bool) and fence_token == 0:
```

Comment:
this is broken (see the comment above) and won't work for `0L`

Reply:
This line had to be modified, at a minimum because `0L` is invalid syntax in Python 3.

The `is 0` stuff threw me because typically you would use `==` when comparing numbers. This was a workaround for `False == 0` — to put the logic in English, "`fence_token` must be the number 0, but we can't allow boolean False which masquerades as 0". I think this change makes the boolean guard clear — and the alternative would probably be more awkward.